app/parsers/pdf.py

```diff
@@ -1,10 +1,22 @@
 from PyPDF2 import PdfReader
-
+from typing import List
+from langchain.document_loaders import PyPDFLoader
+from langchain.schema import Document
+from langchain.text_splitter import CharacterTextSplitter
 def pdf_to_text(input_filename: str) -> str:
 	pdf_file = PdfReader(input_filename)
 	text=''
 	
 	for page in pdf_file.pages:
 		text = text + page.extract_text()
 	
-	return text
\ No newline at end of file
+	return text
+
+
+def pdf_to_textV2(input_filename: str) -> List[Document]:
+	loader = PyPDFLoader(input_filename)
+	documents = loader.load()
+	text_split = CharacterTextSplitter(chunk_size=100, chunk_overlap=0)
+	texts = text_split.split_documents(documents)
```

Comment:
add here the pdf->text, this func should just return str

Reply:
@Roey7 refactored and now `pdf_to_text` fucntion returns a `str`